OIPA/api/middleware.py

```diff
@@ -0,0 +1,37 @@
+from django.urls import resolve
+from django.utils.datastructures import MultiValueDictKeyError
+
+
+class FileExportMiddleware:
```

Comment:
I would also include a docstring here describing exactly what this middleware does and what is the exact  purpose of if. Otherwise (at least for me) while reading the code a question would arrive - why this developer couldn't have simply used Django rest framework's renderers ? Why  do we need this middleware (might sound a bit overkill at the beginning, but we have a reason for usiing middleware ofc.). You know what I mean. Also, documenting code is a good practise